docs/docs/04-configuration/03-auth-providers.md

```diff
@@ -1,2 +1,30 @@
 # Auth Providers
-Include details about how to configure each supported auth provider
\ No newline at end of file
+
+Authentication providers allow your Otto installation to authenticate users with the identity provider of your choice. All authentication providers are configured using environment variables.
+
+## Common Environment Variables
+
+The following environment variables are required for all authentication providers. Setting the Client ID and Client Secret will mean that the authentication provider is enabled. The remaining configuration will be validated on startup.
+
+- `OTTO_AUTH_CLIENT_ID`: The client ID of the authentication provider.
+- `OTTO_AUTH_CLIENT_SECRET`: The client secret of the authentication provider.
+- `OTTO_AUTH_COOKIE_SECRET`: The secret used to encrypt the authentication cookie. Must be of size 16, 24, or 32 bytes.
+- `OTTO_AUTH_ADMIN_EMAILS`: A comma-separated list of the email addresses of the admin users.
+
+The following environment variables are optional for all authentication providers:
+- `OTTO_AUTH_EMAIL_DOMAINS`: The email domains allowed for authentication. Defaults to `*`.
```

Comment:
I assume this is a comma-separated list just like the admin emails, but we may as well be explicit about it as well

Reply:
It is. I changed the wording to make it explicit.